### PR TITLE
[fmt] Coordinates

### PR DESCRIPTION
**Pull request by RMeli**
----
📚 Documentation preview 📚: https://mdanalysis--4856.org.readthedocs.build/en/4856/

### CODE REVIEW

File: testsuite/MDAnalysisTests/coordinates/test_chainreader.py [hunk 1]
```diff
@@ -288,38 +337,68 @@ def __init__(self, seq, n_frames, order):
...
+                order=[2, 2, 1, 1, 1, 0, 0, 0, 0, 0],
+            ),
+            SequenceInfo(seq=([0, 1, 2],) * 3, n_frames=3, order=[2, 2, 2]),
+            SequenceInfo(
+                seq=(
+                    [
+                        0,
+                        1,
+                        2,
+                        3,
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/coordinates/test_chainreader.py]: Trailing comma.

File: testsuite/MDAnalysisTests/coordinates/test_reader_api.py [hunk 2]
```diff
@@ -222,55 +248,78 @@ def test_getitem(self, slice_cls, sl, reader):
...
+            * 10,
+            [
+                False,
+            ]
+            * 10,
+            [
+                True,
+                False,
+            ]
+            * 5,
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/coordinates/test_reader_api.py]: Trailing commas.

File: testsuite/MDAnalysisTests/coordinates/test_reader_api.py [hunk 3]
```diff
@@ -222,55 +248,78 @@ def test_getitem(self, slice_cls, sl, reader):
...
-    ])
+    @pytest.mark.parametrize(
+        "sl1",
+        [
+            [0, 1, 2, 3, 4],
+            [1, 1, 1, 1, 1],
+            [
+                True,
+                False,
+            ]
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/coordinates/test_reader_api.py]: Trailing commas.

File: testsuite/MDAnalysisTests/coordinates/test_reader_api.py [hunk 4]
```diff
@@ -285,15 +334,22 @@ def test_double_getitem(self, sl1, sl2, reader):
...
-    @pytest.mark.parametrize('idx2', [0, 2, 4, -1, -2, -4])
+    @pytest.mark.parametrize(
+        "sl1",
+        [
+            [0, 1, 2, 3, 4],
+            [1, 1, 1, 1, 1],
+            [
+                True,
+                False,
+            ]
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/coordinates/test_reader_api.py]: Trailing commas.

File: testsuite/MDAnalysisTests/coordinates/test_reader_api.py [hunk 5]
```diff
@@ -317,14 +373,21 @@ def sl():
...
-    @pytest.mark.parametrize('idx2', [5, -6])
+    @pytest.mark.parametrize(
+        "sl1",
+        [
+            [0, 1, 2, 3, 4],
+            [1, 1, 1, 1, 1],
+            [
+                True,
+                False,
+            ]
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/coordinates/test_reader_api.py]: Trailing comma.

**Comment by RMeli** [comment]
Fixed trading commas. Good to go IMO.

**Review by orbeckst — APPROVED** [review]
blanket LGTM — please merge when you're happy with it @RMeli 

🎄 🎁

**Comment by marinegor** [comment]
hey @RMeli sorry it took so long on my side -- LGTM too, no big issues.

Perhaps a little suggestion: could you limit the formatting PRs to ~20-25 files each? Petty reason but my browser keeps crushing during the review process :)

**Pull request merged**